Gemfile

```diff
@@ -1,12 +1,13 @@
 source "https://rubygems.org"
 
-# Specify your gem's dependencies in codeclimate-test-reporter.gemspec
-gemspec
+gem "rake"
+gem "rspec"
+gem "webmock"
+gem "pry"
```

Comment:
I'd prefer to keep these dependencies within the gemspec.

Ideally, this file would only contain two lines:

```ruby
source "http://www.rubygems.org"

gemspec
```